xwiki-platform-core/xwiki-platform-refactoring/xwiki-platform-refactoring-default/src/test/java/org/xwiki/refactoring/internal/DefaultModelBridgeTest.java

```diff
@@ -179,6 +180,18 @@ public void create(MockitoComponentManager mocker) throws Exception
         assertLog(Level.INFO, "Document [{}] has been created.", documentReference);
     }
 
+    @Test
+    public void createWithException(MockitoComponentManager mocker) throws Exception
```

Comment:
`public` keyword should be ommitted.

Reply:
mocker is not a good name. I'd use `componentManager`.

Reply:
Renamed (see https://github.com/xwiki/xwiki-platform/pull/1401/files#diff-29fdf5e5e3805db76947ad7e90902ac7R194)